turn/src/allocation/allocation_manager/allocation_manager_test.rs

```diff
@@ -375,3 +382,209 @@ async fn test_delete_allocation_by_username() -> Result<()> {
 
     Ok(())
 }
+
+#[tokio::test]
+async fn test_get_allocations_info() -> Result<()> {
+    let turn_socket: Arc<dyn Conn + Send + Sync> = Arc::new(UdpSocket::bind("0.0.0.0:0").await?);
+
+    let m = new_test_manager(true);
+
+    let five_tuple1 = random_five_tuple();
+    let five_tuple2 = random_five_tuple();
+    let five_tuple3 = random_five_tuple();
+
+    let alloc1 = m
+        .create_allocation(
+            five_tuple1.clone(),
+            Arc::clone(&turn_socket),
+            0,
+            DEFAULT_LIFETIME,
+            TextAttribute::new(ATTR_USERNAME, "user1".into()),
+        )
+        .await?;
+    alloc1.relayed_bytes.store(111, Ordering::SeqCst);
```

Comment:
This way to test this seems kind of shallow, wouldn't it be better to actually send some data and verify it this way?

Reply:
I think it would. I will add tests with sending real data.